assets/src/room_ui.ts

```diff
@@ -26,6 +26,53 @@ type MediaStreams = {
   videoStream: MediaStream | null;
 };
 
+export function addAudioStatusChangedCallback(
+  callback: (status: boolean) => any
+) {
+  audioButton.addEventListener("click", () =>
+    callback(!getAudioButtonStatus())
+  );
+}
+
+export function addVideoStatusChangedCallback(
+  callback: (status: boolean) => any
+) {
+  videoButton.addEventListener("click", () =>
+    callback(!getVideoButtonCurrentStatus())
```

Comment:
Why do we negate current status? Also, let's call this `getVideoButtonStatus` to be consistent with `getAudioButtonStatus`.

Reply:
This function add a callback for a status change. The button doesn't automatically change the state, you need a callback to do it. The negation is effectively status change logic

Reply:
This function add a callback for a status change. The button doesn't automatically change the state, you need a callback to do it. The negation is effectively status change logic